packages/components/src/useChrome/useChrome.js

```diff
@@ -0,0 +1,17 @@
+import { useScalprum } from '@scalprum/react-core';
+
+const useChrome = (selector) => {
+    const state = useScalprum();
+    let chrome = state?.api?.chrome || {};
+    chrome = {
+        ...chrome,
+        initialized: state.initialized
```

Comment:
what if `state` is undefined?

Reply:
It can't be, by default it's always an empty object. It could only be undefined if you were using the hook outside of the context.

Reply:
see line number 5 then 👀

Reply:
:smiling_imp: